moodle-ldap.conf

```diff
@@ -0,0 +1,17 @@
+#
+# LDAP Defaults
+#
+
+# See ldap.conf(5) for details
+# This file should be world readable but not world writable.
+
+#BASE   dc=example,dc=com
+#URI    ldap://ldap.example.com ldap://ldap-master.example.com:666
+
+#SIZELIMIT      12
+#TIMELIMIT      15
+#DEREF          never
+
+# TLS certificates (needed for GnuTLS)
+TLS_CACERT      /etc/ssl/certs/ca-certificates.crt
+TLS_REQCERT     never
```

Comment:
`TLS_RECERT` never might not be a very secure default :-/

Reply:
Leaving it commented out with an explanation that internal / self signed certificates aren’t trusted by default might be a suitable compromise?